### DISCUSSION TITLE
New to Toonsquid….import png?

### DISCUSSION THREAD

**Discussion by MJAlbertSculpts** [keiwando/toonsquid#48] [Q&A]
I’d like to import a png to use as a background. When I navigate to the file I want, it’s darkened and cannot be selected. What am I missing, here?

**Comment by MJAlbertSculpts** [comment]
Never mind, I found the solution. Wow, this is going to be awesome!

**Reply by saramashfej** [reply]
Hi, I’m having the same issue with importing mp4. How were you able to fix this?
